config/locales/views/pages/nl.yml

```
@@ -34,17 +34,52 @@ nl:
       no-activities-help: "Je hebt alle oefeningen opgelost!"
       start-activity: "Begin met %{activity}"
       continue-activity: "Werk verder aan %{activity}"
+      unanswered-questions:
+        one: "Één open vraag"
+        other: "%{count} open vragen"
+      unanswered-questions-subtitle:
+        one: "Klik hier om deze vraag te bekijken"
+        other: "Klik hier om deze vragen te bekijken"
+      pending-members:
+        one: "Één gebruiker wacht op goedkeuring"
+        other: "%{count} gebruikers wachten op goedkeuring"
+      pending-members-subtitle:
+        one: "Klik hier om deze gebruiker te bekijken"
+        other: "Bekijk alle gebruikers die toegang to de cursus vragen"
+      incomplete-feedbacks:
+        one: "Één onbeoordeelde oplossing"
```

I think we use the term "afgewerkte oplossing", so maybe something like this?

```suggestion
        one: "Één onafgewerkte oplossing"
```

The docs also talk about "onbekeken oplossingen", so perhaps that's an alternative.

---

For the same reason as in English, I think 'afgewerkt' does not apply.

I had `onbekeken` before, but it looked rather ugly

---

We hebben dit vanmorgen bespreken. Er stond eerst "onbekeken", maar dat voelde raar aan. Daarom is het aangepast naar "onbeoordeeld" (al kan " niet beoordeeld" voor mij ook). Ik was vergeten dat we vroeger "onafgewerkt" gebruikt hebben, maar ik denk dat dat in de context was dat we eerst enkel evaluaties zonder punten hadden. Voor mij werkt "beoordelen" zowel met als zonder punten geven, dus misschien moeten we die vertaling/docs dan overal aanpassen.